acsoo/templates/project/+project.name+/.bumpversion.cfg.bob

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 1.0.0
+commit = False
+
+[bumpversion:file:.bumpversion.cfg]
+
+[bumpversion:file:acsoo.cfg]
+
+[bumpversion:file:odoo_addons/{{{ project.trigram }}}_all/__openerp__.py]
+
+[bumpversion:file:odoo/addons/{{{ project.trigram }}}_all/__manifest__.py]
```

Comment:
While you are at it can you also add the version numbers in the ribbons?

https://github.com/acsone/acsoo/tree/master/acsoo/templates/project/%2Bproject.name%2B/odoo_addons/server_environment_files

https://github.com/acsone/acsoo/tree/master/acsoo/templates/project/%2Bproject.name%2B/odoo/addons/server_environment_files